pkg/server/server_test.go

```diff
@@ -53,44 +53,6 @@ func TestStringEncode(t *testing.T) {
 	}
 }
 
-func TestMachineConfigToRawIgnition(t *testing.T) {
```

Comment:
This could be a unit test for `appendEncapsulated()` now though right?

Reply:
yep I'm cleaning the branches while still testing this in 4.6, I'll sure add a translated unit